src/hdkey.ts

```diff
@@ -20,12 +19,12 @@ export default class EthereumHDKey {
     return new EthereumHDKey(HDKey.fromExtendedKey(base58Key))
   }
 
-  constructor(private readonly _hdkey?: any) {}
+  constructor(private readonly _hdkey: HDKey) {}
```

Comment:
Why did this change from optional to mandatory?

Reply:
`this._hdkey` is used everywhere in the class, also `any` was not type safe

Reply:
👍